script.js

```diff
@@ -1,26 +1,11 @@
-const ui = {
-    carousel: document.querySelector('.carousel'),
-    slides: document.querySelectorAll('.slide'),
-    slidesCounter: document.querySelector('.slides-counter'),
-    slideIndex: 0
-}
-
-function prevSlide() {
-    if (ui.slideIndex === 0) return;
-    ui.slideIndex--;
-    ui.carousel.style.setProperty('--slide-index', ui.slideIndex);
-    updateCounter();
-}
+"use strict";
 
-function nextSlide() {
-    if (ui.slides.length === (ui.slideIndex + 1)) return;
-    ui.slideIndex++;
-    ui.carousel.style.setProperty('--slide-index', ui.slideIndex);
-    updateCounter();
-}
+import initCarousel from "./js/carousel.js";
+import initCategories from "./js/categories.js";
 
-function updateCounter() {
-    ui.slidesCounter.innerText = `${ui.slideIndex + 1}/${ui.slides.length}`
+function init() {
+    initCarousel();
+    initCategories();
 }
```

Comment:
Not sure if wrapping two lines in a function is useful -- try to just inline the function body ;)